### ISSUE TITLE
Audio/Background Only for "x" Playlist

### ISSUE THREAD

**Issue by SerjSX** [TeamNewPipe/NewPipe#7493]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?

Adding a "Audio Only" or "Background Only" toggle on a playlist's page, probably at the top opposite side where the playlist name and amount of videos are in the playlist is put at.  When toggling this on, whenever you play something from the playlist it automatically plays it in the background or only the audio instead of the video.

#### Why do you want this feature?

This feature would be useful if people have playlists that have songs in it and they want to play whatever is in it audio/background only instead of the video, for whatever reason, mine would be less wifi/mobile data consuming and directly listening to the music instead of waiting for the video to load (useful with users that have slow wifi connection).

**Comment by SameenAhnaf** [comment]
@opusforlife2 This discussion should continue on https://github.com/TeamNewPipe/NewPipe/issues/2610 instead, I think.

**Comment by opusforlife2** [comment]
@SameenAhnaf No need to ping specific people. Whoever is triaging issues will see your comments and take action if needed. 👍

**Comment by SerjSX** [comment]
@SameenAhnaf @opusforlife2 if it isn't a problem I want to keep mine separate, the one Sameen mentioned does mention something similar to mine, however it's mixed up with other suggestions as well. If it isn't a problem I want to keep mine separate, one simple issue opened for one suggestion, would be easier to track down for collaborators as well. However, if you want to mix it up with the other one, I guess it would be fine.

**Comment by SameenAhnaf** [comment]
>the one Sameen mentioned does mention something similar to mine

@SerjSX It does so. See: https://github.com/TeamNewPipe/NewPipe/issues/2610#:~:text=separate%20bookmarked%20playlists%20for%20audio%20mode.

> I want to keep mine separate, one simple issue opened for one suggestion

I only thought that it will be worthy of consideration if any interested dev looked into it. After all, audio mode will have to be dedicated for every single action, not just for playlists.

**Comment by SerjSX** [comment]
> > the one Sameen mentioned does mention something similar to mine
> 
> @SerjSX It does so. The second point asks for so.
> 
> > I want to keep mine separate, one simple issue opened for one suggestion
> 
> I only thought that it will be worthy of consideration if any interested dev looked into it. After all, audio mode will have to be dedicated for every single action, not just for playlists.

You mean audio only enabled for ALL videos played? That wouldn't be a good idea, mine is only for playlists, this way if a person has a music playlist they'll play audio/background only instead of loading the video. Let's not mess it up with another thing hehe! But yes, hopefully someone does take it into consideration in the future, would be a good addition.

**Comment by SameenAhnaf** [comment]
That issue description specifically asks for such bookmarked playlists.

<img src="https://user-images.githubusercontent.com/79245641/144739977-e4a022b7-707d-4885-a573-345ba61ddd0f.jpg" height="300px"/>

**Comment by SerjSX** [comment]
> That issue description specifically asks for such bookmarked playlists.
> 
> <img alt="" height="300px" src="https://user-images.githubusercontent.com/79245641/144739977-e4a022b7-707d-4885-a573-345ba61ddd0f.jpg">

Yes, but wouldn't it be simpler if there was just a toggle on all of the playlist instead of a whole separate section for audio mode playlists? I still would like to keep mine separate, but if an administrator/collaborator agrees the opposite then it's fine I guess. The thread you mentioned has a lot of things all top of each other, mine is just one simple suggestion, if there isn't a problem I want this to be separate.
